### PR TITLE
Updates to the logos section of the landing page

### PR DESCRIPTION
**Pull request by beorn7**
- Remove obsolete entries.
- Use HTTPS everywhere.
- Add Grafana Labs.

In different news, I kind of think the use of logos on the front page is outdated. We don't have to prove anything anymore. An approach to have a separate section of linked case studies similar to kubernetes.io might make more sense. However, that's a longer discussion (topic for the dev summit?), and while the logos are still there, we should keep them up to date.

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -116,39 +116,38 @@ <h1><i class="fa fa-github"></i> Open Source</h1>
...
+          <a href="https://www.crononauta.com/"><img src="assets/company-logos/crononauta.png"/></a>
           <a href="https://www.digitalocean.com/"><img src="assets/company-logos/digitalocean.png"/></a>
           <a href="https://www.docker.com/"><img src="assets/company-logos/Docker.png"/></a>
-          <a href="http://www.ericsson.com/"><img src="assets/company-logos/Ericsson.png"/></a>
-          <a href="http://www.eurotech.com/"><img src="assets/company-logos/Eurotech.png"/></a>
+          <a href="https://www.ericsson.com/"><img src="assets/company-logos/Ericsson.png"/></a>
+          <a href="https://www.eurotech.com/"><img src="assets/company-logos/Eurotech.png"/></a>
           <a href="https://www.fosdem.org/"><img src="assets/company-logos/Fosdem.png"/></a>
           <a href="https://giantswarm.io/"><img src="assets/company-logos/giantswarm.png"/></a>
+	  <a href="https://grafana.com"><img src="assets/company-logos/grafana.png"/></a>
```
Comment (juliusv) [inline content/index.html]: This one's indented inconsistently.

**Comment by juliusv** [comment]
:+1: besides nit

File: content/index.html (hunk 1 above)
Comment (beorn7) [reply content/index.html]: My naughty Emacs introduced a tab here. Untabify helped…

**Comment by juliusv** [comment]
:+1:

**Comment by brian-brazil** [comment]
> In different news, I kind of think the use of logos on the front page is outdated. 

The reason for them originally being there no longer applies, so I think it would make sense to make them less prominent.

**Pull request merged**